examples/mutators/set-annotation/advanced/README.md

```diff
@@ -0,0 +1,44 @@
+# Set Annotation Advanced Example
```

Comment:
`set-annotation` is a proper noun, so try to refer to it consistency instead of splitting it into different words.

`set-annotation: Advanced Example`